.changeset/dry-cats-change.md

```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/backend': minor
+---
+
+New API (experimental):
+Introducing `AccountlessApplicationAPI` accessible via `clerkClient.__experimental_accountlessApplications`.
+It allows for an Accountless application to be generated and response with a publishable key, secret key and a URL that someone can use in order to claim the application in their account. 
```

Comment:
New **experimental** API: `AccountlessApplicationAPI`

Inside `clerkClient` you can activate this new API through `__experimental_accountlessApplications`. It allows you to generate an "accountless" application and returns the publishable key, secret key, and an URL as a response. The URL allows a user to claim this application with their account. Hence the name "accountless" because in its initial state the application is not attached to any account yet.